build/symlink-dependencies/index.js

```diff
@@ -49,7 +49,8 @@ module.exports = {
         if (isScopedPackage(dep)) { mkdirpScope(dep, modulesPath); }
         fs.symlinkSync(
           path.join('../../../../', dep),
-          path.join(modulesPath, dep)
+          path.join(modulesPath, dep),
+          'junction'
```

Comment:
Is this conditional? In other words, do we have to detect the platform first and do one thing for windows and another for *nix?

Reply:
I believe it is ignored outside of Windows, and it passed CI so I would assume that makes it okay.